### DISCUSSION TITLE
Closing Session

### DISCUSSION THREAD

**Discussion by Amadou507** [cybertec-postgresql/pg_timetable#636] [General]
I installed pg_timetable according to the steps from the official site (first, according to the official release packages and then according to the build from sources). After doing it, I scheduled the requests as a test but I noticed that every time I exit my terminal, pg_timetable logs off and on top of that, I have to work on 2 pages of the terminal (the first for recording requests in postgres and the second for keeping a session open). If I don't do it like this, the queries won't execute.

**Comment by pashagolub** [comment]
Please make yourself familiar on how to run applications as background processes on the operating system of your choice. If you're on Linux you should [daemonize a process or use systemd](https://baykara.medium.com/how-to-daemonize-a-process-or-service-with-systemd-c34501e646c9), for example. If you're on Windows, [there are some options](https://answers.microsoft.com/en-us/windows/forum/all/starting-applications-in-the-background/300d5658-9a0d-44a8-b2e6-7faf3d3632f7) as well.

**Reply by Amadou507** [reply]
I'm on a Linux system
